pkg/http/api.go

```diff
@@ -626,15 +626,16 @@ func (a *api) onBulkGetSecret(reqCtx *fasthttp.RequestCtx) {
 		return
 	}
 
-	for key := range resp.Data {
-		if !a.isSecretAllowed(secretStoreName, key) {
-			msg := NewErrorResponse("ERR_PERMISSION_DENIED", fmt.Sprintf(messages.ErrPermissionDenied, key, secretStoreName))
-			respondWithError(reqCtx, fasthttp.StatusForbidden, msg)
-			return
+	filteredSecrets := map[string]string{}
+	for key, v := range resp.Data {
+		if a.isSecretAllowed(secretStoreName, key) {
+			filteredSecrets[key] = v
+		} else {
+			log.Warnf(messages.ErrPermissionDenied, key, secretStoreName)
```

Comment:
I don't think we should log a warning here. Only returning the allowed secrets is the expected result.

Reply:
See: https://github.com/dapr/dapr/pull/2624#discussion_r551785441